numpy/core/tests/test_einsum.py

```diff
@@ -605,6 +605,11 @@ def test_einsum_misc(self):
                      [[[1,  3], [3,  9], [5, 15], [7, 21]],
                      [[8, 16], [16, 32], [24, 48], [32, 64]]])
 
+        # Ensure explicitly setting out=None does not cause an error
+        # see issue #15776 and issue #15256
```

Comment:
Future-proofing in case we ever switch issue tracker again:
```suggestion
        # see gh-15776 and gh-15256
```

Reply:
Alright thank you, will also update that :+1: